docs/JLBP-0006.md

```diff
@@ -82,8 +82,8 @@ Given this scenario, here are the possible combinations of renamings:
     will only choose one of them to use. Users are forced to change their code
     between versions, but only for library surface that was changed.
 
-  - **Case 2: Rename Maven ID**. Users can easily pull in both the old jar
-    (`g1:a1:1.0.0`) and the new jar (`g2:a2:2.0.0`) into their classpath
+  - **Case 2: Rename Maven ID**. Users can easily pull in both the old artifact
+    (`g1:a1:1.0.0`) and the new artifact (`g2:a2:2.0.0`) into their classpath
```

Comment:
A JAR file does not have groupID or artifactID. It's a Maven artifact that has groupID and artifactID. A Maven artifact usually has a JAR file (except for pom-type artifacts).

Reply:
but only JARs are added to classpaths

Reply:
That's true. Updated.